scripts/web/blueprints/reports.py

```
@@ -0,0 +1,134 @@
+                    "overall_score":    g.get("overall_score", ""),
+                    "grading_model":    g.get("grading_model", ""),
+                    "confidence":       g.get("confidence", ""),
+                    "overall_feedback": g.get("overall_feedback", ""),
+                }
+                for cs in criterion_scores:
+                    cid = cs.get("criterion_id", cs.get("criterion_name", "?"))
+                    row[f"{cid}_awarded"] = cs.get("awarded_points", "")
+                    row[f"{cid}_max"]     = cs.get("max_points", "")
+                rows.append(row)
+            except Exception:
+                pass
```

Ruff (0.15.11)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)